client/gutenberg/extensions/prev-next/index.js

```diff
@@ -0,0 +1,75 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import wp from 'wp';
+const { __ } = wp.i18n;
+const { TextControl } = wp.components;
+const { registerBlockType } = wp.blocks;
+
+require( './style.scss' );
+
+const blockAttributes = {
+	prev: {
+		type: 'string',
+		source: 'attribute',
+		selector: 'a:first-child',
+		attribute: 'href',
+	},
+	next: {
+		type: 'string',
+		source: 'attribute',
+		selector: 'a:last-child',
+		attribute: 'href',
+	},
+};
+
+const save = ( { attributes: { prev, next }, className, isEditor } ) =>
+	prev || next ? (
+		<div className={ isEditor ? className : '' }>
+			{ prev ? <a href={ prev }>← Prev</a> : <span> </span> }
+			{ next ? <a href={ next }>Next →</a> : <span> </span> }
+		</div>
+	) : (
+		<React.Fragment />
+	);
+
+registerBlockType( 'a8c/prev-next', {
+	title: __( 'Prev/Next Links' ),
+	icon: 'leftright',
+	category: 'common',
+	description: __( 'Link this post to sequential posts in a series of related posts.' ),
+	keywords: [ __( 'links' ) ],
+	attributes: blockAttributes,
+	edit: ( { attributes, className, isSelected, setAttributes } ) => {
+		if ( isSelected ) {
```

Comment:
👍 I like the changes you've done here, they make `edit()` more readable IMHO.

Reply:
this was all tossed together very quickly in order to supply a specific functionality in our p2 posts. part of the point was "look how quickly one can build a block" and part was "I gots me other more important stuffs to do"

🙃